modules/ui/CaptionUI.py

```diff
@@ -82,6 +82,7 @@ def __init__(self, parent, initial_dir: str | None, *args, **kwargs):
         self.image = None
         self.image_label = None
         self.enable_mask_editing_var = ctk.BooleanVar()
+        self.enable_mask_editing_alpha_var = None
```

Comment:
The name `enable_mask_editing_alpha_var` doesn't really make sense here. `mask_editing_alpha_entry` would be better.

Reply:
Done